label_studio/core/utils/iterators.py

```diff
@@ -0,0 +1,33 @@
+from core.feature_flags import flag_set
+from django.conf import settings
+
+
+def iterate_queryset(queryset, chunk_size=None):
+    if chunk_size is None:
+        chunk_size = settings.QS_ITERATOR_DEFAULT_CHUNK_SIZE
+
+    if chunk_size <= 0:
+        raise ValueError(f'chunk_size must be positive, got {chunk_size}')
+
+    if not flag_set('fflag_fix_back_plt_863_remove_iterator_27082025_short', user='auto'):
```

Comment:
do we need a FF for this? Why would we ever turn off the FF and use normal iterator? On prem I guess?

Reply:
I added it for the case, if a change in query patterns will cause some unexpected database load, to be able to quickly revert the change and avoid long downtime. It can be removed after release, if everything is fine.